test/SparqlExpressionTest.cpp

```diff
@@ -702,8 +702,22 @@ auto checkLcase = testUnaryExpression<&makeLowercaseExpression>;
 TEST(SparqlExpression, uppercaseAndLowercase) {
   checkLcase(IdOrLiteralOrIriVec{lit("One"), lit("tWÖ"), U, I(12)},
              IdOrLiteralOrIriVec{lit("one"), lit("twö"), U, U});
+  checkLcase(
+      IdOrLiteralOrIriVec{
+          lit("One", "^^<http://www.w3.org/2001/XMLSchema#string>"),
+          lit("One", "@en"), U, I(12)},
+      IdOrLiteralOrIriVec{
+          lit("one", "^^<http://www.w3.org/2001/XMLSchema#string>"),
+          lit("one", "@en"), U, U});
   checkUcase(IdOrLiteralOrIriVec{lit("One"), lit("tWÖ"), U, I(12)},
              IdOrLiteralOrIriVec{lit("ONE"), lit("TWÖ"), U, U});
+  checkLcase(
```

Comment:
This fails because it should probably be `checkUcase`.
Please also try out your builds and unit tests on your own machine before pushing them, because running all the CI checks takes quites some time and resources also from other contributors.